sky/engine/core/script/dart_init.cc

```diff
@@ -420,7 +430,7 @@ void InitDartVM() {
   }
   CHECK(Dart_SetVMFlags(args.size(), args.data()));
 
-  {
+  if (IsDevelopMode()) {
```

Comment:
Does this compile? I would expect DartDebugger::InitDebugger() to not be linked based on your above gn chagnes?